packages/shared-components/src/models/dropdown-option.ts

```diff
@@ -0,0 +1,63 @@
+export class DropdownOption {
+  private _label: string;
+  private _labelKey: string;
+  private _tooltip: string = '';
+  private _tooltipLabelKey: string;
+  private _iconClass: string;
+  private _value: any;
```

Comment:
Why `any`? Isn't this just a plain string?

Reply:
Change it to string